src/interned.rs

```diff
@@ -896,6 +897,20 @@ where
         VerifyResult::unchanged()
     }
 
+    fn minimum_serialized_edges(&self, _zalsa: &Zalsa, edge: QueryEdge) -> Vec<QueryEdge> {
+        if C::PERSIST {
```

Comment:
Interesting: I initially assumed that an implementation flattening the dependency at runtime would use the same logic as what we need for persistence, but I can see now that separate methods make sense.